openhtf/util/validators.py

```diff
@@ -5,7 +5,7 @@
 module, and will typically be a type, instances of which are callable:
 
   from openhtf.util import validators
-  from openhtf.util import measurements
+  from openhtf.util.validators import ValidatorBase
```

Comment:
Our style guide does not recommend importing classes and functions from modules.

from openhtf.util import validators